src/Apps/Legion/Components/WizardShell/Internal/ModifyStep/Internal/DiagramTab/DiagramTab.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { TDiagramTabClassNames } from './DiagramTab.types';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+
+export const useClassNames = makeStyles<TDiagramTabClassNames>({
+    root: {},
+    diagramSelector: {
+        width: '308px',
```

Comment:
for picker dropdowns, we have been using a fixed width for either itself or for the parent component when consuming it. Because we want to make sure to show the content of the items properly enough and not to take much space. I can set it to a percentage instead here like 40% but it will unnecessarily take up space when the item content is not that long. I can move it to constant, but I think I prefer it to live in the types file of that component because that constant is only for this component.

Reply:
yeah. I guess for dropdowns lets use fixed amounts. I had seen widths in a few other spots and just commented here.